test/Microsoft.ML.FSharp.Tests/Microsoft.ML.FSharp.Tests.fsproj

```diff
@@ -8,7 +8,7 @@
     <NoWarn>2003;$(NoWarn)</NoWarn>
     <PublicSign>false</PublicSign>
     <SourceLink></SourceLink>
-    <PlatformTarget>x64</PlatformTarget>
+    <PlatformTarget>$(TargetArchitecture)</PlatformTarget>
```

Comment:
Can this just be left to the default Any CPU?

Reply:
No, removing it, leaving it empty, and setting it to `AnyCPU` causes the tests to fail on x64